p2p/host.go

```diff
@@ -141,16 +130,20 @@ func NewHost(cfg HostConfig) (Host, error) {
 	)
 
 	ctx, cancel := context.WithCancel(context.Background())
-	// TODO: move low and high to configs
-	connmgr, err := connmgr.NewConnManager(
-		int(cfg.MaxConnPerIP),      // LowWater
```

Comment:
is LowWater/HighWater, differentiate a host IP that has multiple instance of the harmony binary with p2p ID